examples/embedding/Makefile

```diff
@@ -1,8 +1,17 @@
 MPTOP = ../..
+include $(MPTOP)/py/mkenv.mk
+
 CFLAGS = -std=c99 -I. -I$(MPTOP) -DNO_QSTR
 LDFLAGS = -L.
 
 hello-embed: hello-embed.o -lmicropython
 
 -lmicropython:
 	$(MAKE) -f $(MPTOP)/examples/embedding/Makefile.upylib MPTOP=$(MPTOP)
+
+clean:
+	$(MAKE) -f $(MPTOP)/examples/embedding/Makefile.upylib MPTOP=$(MPTOP) clean
+	$(RM) -f hello-embed.o
+	$(RM) -f hello-embed
+
+all: hello-embed
```

Comment:
Missing newline (I'm lookin at this on github and that makes it easy to see that; maybe configure your text editor to automatically add newline at end of file?). For the rest this is all good I think.

Reply:
Its a strange things in vs code, for boardconfig I manually placed two empty lines to make it looks like one empty line in real file...

Reply:
fixed